src/components/image-with-caption/image-with-caption.vue

```diff
@@ -28,10 +32,17 @@ defineProps<{
     alt?: string,
     sizes: string,
   },
-}>()
+  imagePosition: string
```

Comment:
Could we make this:

```suggestion
  imagePosition: 'left' | 'right' | 'top'
```

Then it's clear what options are possible.